doc/source/security.md

```diff
@@ -155,3 +155,44 @@ This is a sensitive security issue (similar to writing sudo rules in a
 traditional computing environment), so be very careful.
 
 There's ongoing work on making this easier!
+
+
+## Kubernetes Network Policies
+
+Kubernetes has optional support for [network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) which let you restrict how pods can communication with each other and the outside world. This can provide additional security within JupyterHub, and can also be used to limit network access for users of JupyterHub.
+
+Your Kubernetes cluster may have partial, full, or no support for network policies, and Kubernetes will **silently ignore** policies that aren't supported. If you rely on a policy to restrict what users can access you should verify the policies are enforced on your cluster.
+
+Network policies are disabled by default, you can enable them in your `config.yaml`:
+
+```yaml
+hub:
+  networkPolicy:
+    enabled: true
+proxy:
+  networkPolicy:
+    enabled: true
+singleuser:
+  networkPolicy:
+    enabled: true
+```
+
+The default singleuser policy allows all outbound network traffic. To restrict outbound traffic to DNS, HTTP and HTTPS:
```

Comment:
I'm currently commenting on a field where I have little knowledge, but that is a common point of view as well to consider :D 

Perhaps you could mention something about the default network policies for hub/proxy/singleuser as well as kubernetes?

Also have a question myself based on not being read up about network policies, but when you say "outbound traffic", does that mean "cluster outbound" or "pod outbound"? It could be helpful for a person like me to comprehend a difference there if there are.

Reply:
I've expanded the line about outbound singleuser policies. I'm unsure what to add for hub and proxy though since I at the moment I think it's more relevant for developers of this chart, and I don't want to clutter the doc with too much information.